modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/supplemental_claims_controller.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+require 'json_schema/json_api_missing_attribute'
+require 'appeals_api/form_schemas'
+
+class AppealsApi::V2::DecisionReviews::SupplementalClaimsController < AppealsApi::ApplicationController
+  include AppealsApi::JsonFormatValidation
+  # include AppealsApi::StatusSimulation
+  include AppealsApi::CharacterUtilities
+  include AppealsApi::CharacterValidation
+
+  skip_before_action :authenticate
+  before_action :validate_characters, only: %i[create]
+  before_action :validate_json_format, if: -> { request.post? }
+  before_action :validate_json_schema, only: %i[create]
+
+  FORM_NUMBER = '200995'
+  MODEL_ERROR_STATUS = 422
+  HEADERS = JSON.parse(
+    File.read(
+      AppealsApi::Engine.root.join('config/schemas/v2/200995_headers.json')
+    )
+  )['definitions']['supplementalClaimParams']['properties'].keys
+  SCHEMA_ERROR_TYPE = Common::Exceptions::DetailedSchemaErrors
+
+  def create
+    sc = AppealsApi::SupplementalClaim.new(
+      auth_headers: request_headers,
+      form_data: @json_body,
+      source: request_headers['X-Consumer-Username'],
+      api_version: 'V2'
+    )
+
+    render_model_errors(sc) and return unless sc.validate
+
+    sc.save
+
+    render json: AppealsApi::SupplementalClaimSerializer.new(sc).serializable_hash
+  end
+
+  def show
+    id = params[:id]
+    sc = AppealsApi::SupplementalClaim.find(id)
+
+    render json: AppealsApi::SupplementalClaimSerializer.new(sc).serializable_hash
+  rescue ActiveRecord::RecordNotFound
+    render_supplemental_claim_not_found(id)
+  end
+
+  private
+
+  def validate_json_schema
+    validate_json_schema_for_headers
+    validate_json_schema_for_body
+  rescue JsonSchema::JsonApiMissingAttribute => e
```

Comment:
This won't ever get thrown, since we've set the SCHEMA_ERROR_TYPE to DetailedSchemaErrors